src/v2/Apps/Auctions/AuctionsApp.tsx

```diff
@@ -16,7 +16,29 @@ const AuctionsApp: React.FC<AuctionsAppProps> = ({ children }) => {
   return (
     <AppContainer>
       <AuctionsMeta />
-
+      <Box mt={3} ml={4}>
+        <Text variant="largeTitle">Auctions</Text>
+        <Text pt="15px" pb="15px">
+          Bid on thousands of new works every week in leading online auctions.
+        </Text>
+        <Link
+          href="https://www.artsy.net/how-auctions-work"
+          underlineBehavior="hover"
+        >
```

Comment:
Whenever we need a link in force we'll want ot use our `RouterLink` component. And the full url isn't needed; can do `/how-auctions-work`. That way, when we're looking at our work on `staging.artsy.net`, it wont redirect to `www.artsy.net` when we test clicking the button.

Reply:
Redone with use of `RouterLink`